doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,589 @@
+  * Updated samples with support for the :ref:`zephyr:thingy53_nrf5340`.
+  * Fixed an issue where beacons were stopped being sent after node reset.
+  * Fixed an issue where the IV update procedure could be started immediately after the device has been provisioned.
+  * Fixed multiple issues in the :ref:`bt_mesh_sensor_types_readme` module.
+
+* Migration:
+
+  * The model opcode callback :c:member:`bt_mesh_model_op.func` is changed to return an error code if the message processing has failed.
+    If you have implemented your own models, make sure to update opcode handlers of those models.
+  * :ref:`bt_mesh_scene_srv_readme` now extends :ref:`bt_mesh_dtt_srv_readme`.
+    If you are using the Scene Server, make sure that the Generic Default Transition Time Server instance is present on the element that is equal to or lower than the Scene Server's element.
+  * :ref:`bt_mesh_light_hsl_srv_readme` and :ref:`bt_mesh_light_xyl_srv_readme` no longer instantiate the :ref:`bt_mesh_lightness_srv_readme` through :c:macro:`BT_MESH_MODEL_LIGHT_HSL_SRV` and :c:macro:`BT_MESH_MODEL_LIGHT_XYL_SRV` macros respectively.
+    Macros :c:macro:`BT_MESH_LIGHT_XYL_SRV_INIT` and :c:macro:`BT_MESH_LIGHT_HSL_SRV_INIT` now take a pointer to the :c:struct:`bt_mesh_lightness_srv` instance instead.
+    Make sure to instantiate the Light Lightness Server if you are using any of these models.
+
+Matter
+------
+
+* Added:
+
+  * :ref:`Thingy:53 weather station <matter_weather_station_app>` application.
+  * :ref:`Template <matter_template_sample>` sample with a guide about :ref:`ug_matter_creating_accessory`.
+  * :ref:`ug_matter_tools` page with information about building options for Matter controllers.
+  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Matter.
+  * :doc:`Matter documentation set <matter:index>` based on the documentation from the Matter submodule.
+
+Thread
+------
+
+* Added:
+
+  * Added support for nRF21540 for nRF52 Series and nRF53 Series, including Bluetooth® LE in the multiprotocol configuration.
+  * Improvements in :ref:`thread_ug_supported_features_v12`:
+
+    * Thread 1.2 supported in all samples.
+    * Retransmissions are now supported when transmission security is handled by the radio driver.
+    * Support for CSL Accuracy TLV in the MLE Parent Response.
+    * Link Metrics data is now properly updated when using ACK-based Probing.
+
+  * Added support for Thread Backbone Border Router on the :ref:`thread_architectures_designs_cp_rcp` architecture.
+
+* Updated:
+
+  * :ref:`ot_cli_sample` sample - Updated with the USB support.
+  * :ref:`ot_coprocessor_sample` sample - Updated with the USB support.
+  * Removed ``NET_SHELL`` from the Thread samples due to its limited usefulness.
+
+Zigbee
+------
+
```

Comment:
Add this information: "In this release, Zigbee is supported for development and should not be used for production. nRF Connect SDK v1.5.1 contains the certified Zigbee solution supported for production."